src/explore-education-statistics-admin/src/pages/users/components/ReleaseAccessForm.tsx

```diff
@@ -0,0 +1,130 @@
+import { User, UserReleaseRole } from '@admin/services/userService';
+import Button from '@common/components/Button';
+import ButtonText from '@common/components/ButtonText';
+import { FormFieldSelect, FormFieldset } from '@common/components/form';
+import Form from '@common/components/form/Form';
+import { IdTitlePair } from 'src/services/types/common';
+import { Formik, FormikHelpers } from 'formik';
+import orderBy from 'lodash/orderBy';
+import React from 'react';
+
+export interface AddReleaseRoleFormValues {
+  selectedReleaseId: string;
+  selectedReleaseRole: string;
+}
+
+interface Props {
+  releases: IdTitlePair[] | undefined;
```

Comment:
I think we use the following syntax for possibly-undefined properties instead of `string[] | undefined;` : `releaseRoles?: string[]`

Reply:
I've changed that now